spec/lib/submission_runners/cpp_spec.rb

```diff
@@ -0,0 +1,142 @@
+require 'rails_helper'
+require_relative '../../../lib/docker/helpers'
+include Docker::Helpers
+
+RSpec.describe SubmissionRunners::Cpp, type: 'docker' do
+  describe 'docker command and barebone C++ image' do
+    let(:dir) { Dir.pwd + '/spec/fixtures/submission_runners/cpp/' }
+
+    before { @keep_files = Dir.entries(dir) }
+
+    after {
+      compiled = Dir.entries(dir) - @keep_files
+      compiled.each do |file|
+        FileUtils.rm File.join(dir, file)
+      end
+    }
+
+    let(:submission) {
+      ->i, o, e {
```

Comment:
Oh, ok. That's a simple fix. Take a look at some other tests to see how stuff's getting set up.